src/gcp/cloudfunctions.ts

```diff
@@ -559,6 +560,9 @@ export function endpointFromFunction(gcfFunction: CloudFunction): backend.Endpoi
     );
   }
   endpoint.codebase = gcfFunction.labels?.[CODEBASE_LABEL] || projectConfig.DEFAULT_CODEBASE;
+  if (gcfFunction.labels?.[HASH_LABEL]) {
+    endpoint.hash = gcfFunction.labels?.[HASH_LABEL];
```

Comment:
Don't we also need the reverse logic to add the hash label to the function from the endpoint? Look inside `functionFromEndpoint()` on L527 for examples of how we do that with the other label types

Reply:
+1. Is it being done in another PR?

Reply:
This should implement functionFromEndpoint and endpointFromFunction in both cloudfunctions.ts and cloudfunctionsv2.ts

Reply:
Updated! Thanks for calling out!